cfme/scripting/link_config.py

```diff
@@ -51,13 +63,29 @@ def main(src, dest):
     for element in filter(_is_yaml_file, src.iterdir()):
         _warn_on_unknown_encryption(element)
         target = dest.joinpath(element.name)
-        # the following is fragile
-        if target.is_symlink():
-            _warn_on_missmatching_symlink(src=element, target=target)
-        elif target.is_file():
-            _warn_on_existing_file(target)
+
+        if force:
+            try:
+                target.symlink_to(element.resolve())
+            except OSError as e:
+                if e.errno == errno.EEXIST:
+                    backup_target = Path(dest.joinpath(element.name + "_bak"))
+                    print("Replacing", target.name, "and saving backup as", backup_target.name)
+                    # Would use 'backup_target.replace()' here but that's only supported in py3
+                    if backup_target.exists():
+                        os.remove(str(backup_target))
+                    target.rename(backup_target)
+
+                    target.symlink_to(element.resolve())
+                else:
+                    raise e
```

Comment:
**OPTIONAL**: Just `raise`, that will keep the original traceback, this one would mask it.